game-core/src/main/java/games/strategy/engine/data/GameData.java

```diff
@@ -59,8 +58,7 @@
  */
 public class GameData implements Serializable {
   private static final long serialVersionUID = -2612710634080125728L;
-  private final ReadWriteLock readWriteLock = new ReentrantReadWriteLock();
-  private transient LockUtil lockUtil = LockUtil.INSTANCE;
+  private ReadWriteLock readWriteLock = new ReentrantReadWriteLock();
```

Comment:
Since you are always setting this field in `readObject()`, can it be marked `transient`?

Reply:
Good point, will do